### PR TITLE
release version 0.37.0

### PR DESCRIPTION
**Pull request by seedlit**
What's new:
 - Fixed inadvertent titles and tags removals during asset metadata updates.
 - Dropped unneeded `auth::env` property and the corresponding tests.
 - Generalized new authentication stack to cover account authentication case.
 - Added new components within the HTTP layer to facilitate future enhancements in authentication and request processes.
 - Adjusted most of the code in accordance with pylint checks.

Items:
* [ ] Ran test & live-tests
* [ ] Implemented (new) unit tests
* [ ] Removed credentials
* [ ] Removed argument pointing to staging
* [ ] Updated [documentation](sdk.up42.com)

For release:
* [x] Bumped version
* [x] Added changelog
* [x] Updated announcement banner

### CODE REVIEW

File: docs/theme_override_home/main.html [hunk 1]
```diff
@@ -20,7 +20,7 @@
 <!-- Announcement bar -->
 {% block announce %}
   <style>.md-announce a,.md-announce a:focus,.md-announce a:hover{color:currentColor}.md-announce strong{white-space:nowrap}.md-announce .twitter{margin-left:.2em;color:#00acee}</style>
-    <strong><a href="https://sdk.up42.com/CHANGELOG/">New in 0.36.0: Ordering and estimation functions are updated</a></strong>
+    <strong><a href="https://sdk.up42.com/CHANGELOG/">New in 0.37.0: Fixed inadvertent titles and tags removals during asset metadata updates</a></strong>
```
Comment (javidq) [inline docs/theme_override_home/main.html:23]: We need to change this to `https://docs.up42.com/sdk/changelog`

File: CHANGELOG.md [hunk 2]
```diff
@@ -29,6 +29,18 @@ You can check your current version with the following command:
     ```
 
 For more information, see [UP42 Python package description](https://pypi.org/project/up42-py/).
+
+## 0.37.0
+
+**March 15, 2024**
+
+- Release that accommodates following changes:
```
Comment (javidq) [inline CHANGELOG.md:37]: - is not needed here

File: docs/theme_override_home/main.html (hunk 1 above)
Comment (javidq) [reply docs/theme_override_home/main.html:23]: titles -> title

**Review by javidq — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -29,6 +29,18 @@ You can check your current version with the following command:
     ```
 
 For more information, see [UP42 Python package description](https://pypi.org/project/up42-py/).
+
+## 0.37.0
```
Comment (dustgalactic) [inline CHANGELOG.md:33]: FYI, will be published here: https://github.com/up42/documentation-hub/pull/1144/files

**Review by javidq — APPROVED** [review]

**Review by dustgalactic — APPROVED** [review]

**Pull request merged**